nengo_viz/static/viz_netgraph.js

```diff
@@ -114,7 +114,9 @@ VIZ.NetGraph = function(parent, args) {
             var x = (event.clientX / $(self.svg).width())
             var y = (event.clientY / $(self.svg).height());
 
+            self.normalize_mousewheel(event);
             var delta = event.wheelDeltaY || -event.deltaY
+            VIZ.scale.step_size = 1.05
             var scale = delta > 0 ? VIZ.scale.step_size : 1.0 / VIZ.scale.step_size; // will either be 1.1 or ~0.9
```

Comment:
I'm confused...  it looks to me like ```normalize_mousewheel``` does some wacky math in order to compute a nice ```event.delta```, which we then throw away because this ```scale``` calculation completely ignores it.